### ISSUE TITLE
Adding emoji feature

### ISSUE THREAD

**Issue by saketkumar95** [fossasia/susi_android#462]
**Actual Behaviour**

Currently there is no option to send emojis in the app.

**Expected Behaviour**

It would be better to implement this. As emoji feature are the base of these types of chat app.

**Would you like to work on the issue?**

Yes i would like to work on this feature.

**Comment by Abhi2424shek** [comment]
@saketkumar95 updates?

**Comment by daminisatya** [comment]
@saketkumar95 What is your aproach? We need to bring up the support from susi server, frame rules for accepting the emoticon characters. I guess you can raise an issue there and discuss the further steps.

**Comment by saketkumar95** [comment]
@daminisatya yeah sure. I will raise the issue in the server too. If you allow i can work on the server too for this issue. And in susi android we can use any open source library. :)

**Comment by daminisatya** [comment]
@saketkumar95 Sure you can work!

**Comment by jig08** [comment]
>I guess you can raise an issue there and discuss the further steps.

@daminisatya @saketkumar95 There exists an issue on this already, check out Issue https://github.com/fossasia/susi_server/issues/3. Avoid duplicate issue. Thanks !
